examples/camera-controller/commands/interactive/Commands.h

```diff
@@ -28,6 +28,7 @@ void registerCommandsInteractive(Commands & commands, CredentialIssuerCommands *
 
     commands_list clusterCommands = {
         make_unique<InteractiveStartCommand>(&commands, credsIssuerConfig),
+        make_unique<InteractiveServerCommand>(&commands, credsIssuerConfig),
```

Comment:
Why put this command under 'interactive'?

"interactive server" sounds strange, should it be under 'websocket', and use command "websocket start" to start the websocket?

Reply:
Thank you for your comments. Basically, TH(Test Harness) is working with chip-tool as of now. TH is using interactive server command. If we want to use camera-controller on TH. We should provide this command. Thank you.